client/src/featureform/register_test.py

```diff
@@ -100,15 +130,52 @@ def test_register_user(registrar, args, expected):
 ])
 def test_register_dynamodb(registrar, args):
     registrar.register_dynamodb(**args)
-
-
+    
 minimal_user_args = {
     "name": "user",
     "region": "abc",
     "access_key": "abc",
     "secret_key": "abc"
 }
 
+
+@pytest.mark.parametrize("args,expected", [(minimal_user_args, User("user"))])
+def test_register_user(registrar, args, expected):
```

Comment:
what's the need for the two of the same test? Is it the same as Lines 112-117? If so, can we combine them?